packages/simcore-sdk/src/simcore_sdk/node_ports_common/aws_s3_cli_utils.py

```diff
@@ -13,7 +13,7 @@
 def _parse_size(log_string):
     match = re.search(r"^\w+ (?P<size>[^\/]+)", log_string)
     if match:
-        return match.group("size")
+        return match.group("size").replace("Bytes", "")
```

Comment:
MIONR: can you add a small doc why the word "Bytes" appears here? perhaps a link to this PR would suffice since you explain some of ti?